extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -547,7 +547,12 @@ class AmpCarousel extends AMP.BaseElement {
 
     const action = createCustomEvent(this.win, `slidescroll.${name}`, data);
```

Comment:
Since `data` isn't being used below anymore, it's fine to place its value here in the parameter instead of declaring it a few lines above.